code/modules/research/techweb/all_nodes.dm

```diff
@@ -97,13 +97,13 @@
 	design_ids = list("surgery_lobotomy", "surgery_heal_brute_upgrade_femto","surgery_heal_burn_upgrade_femto","surgery_heal_combo", "surgery_revival","surgery_adv_dissection")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1500)
 	export_price = 4000
-  
+
 /datum/techweb_node/exp_surgery
 	id = "exp_surgery"
 	display_name = "Experimental Surgery"
 	description = "When evolution isn't fast enough."
 	prereq_ids = list("adv_surgery")
-	design_ids = list("surgery_pacify","surgery_vein_thread","surgery_muscled_veins","surgery_nerve_splice","surgery_nerve_ground","surgery_ligament_hook","surgery_ligament_reinforcement","surgery_viral_bond", "surgery_heal_combo_upgrade", "surgery_exp_dissection")
+	design_ids = list("surgery_pacify","surgery_vein_thread","surgery_muscled_veins","surgery_nerve_splice","surgery_nerve_ground","surgery_ligament_hook","surgery_ligament_reinforcement","surgery_viral_bond", "surgery_heal_combo_upgrade")
```

Comment:
https://github.com/tgstation/tgstation/blob/master/code/modules/research/designs/medical_designs.dm#L586

wut

Reply:
that's not a techweb node

Reply:
Right... It's a design ID that goes in the design ID list (that you're removing here) unless i'm missing something.

Reply:
`## WARNING: Invalid research design with ID surgery_exp_dissection detected in node Experimental Surgery[exp_surgery] removed. in code/controllers/subsystem/research.dm at line 176 src: Research usr: .`

Reply:
the id is `"surgery_ext_dissection"`

Reply:
oh i see what you did